src/react/TextArea/__test__/onEvent.test.tsx

```diff
@@ -0,0 +1,42 @@
+import {render, fireEvent} from '@testing-library/react';
+import TextArea from '../index';
+import React from 'react';
+
+describe('Unit test TextArea react', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => { });
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('onChange with props invisible TextArea', () => {
```

Comment:
This case should be `render TextArea with isVisible = true` and tested in `render` group, not `onEvent`.